std/stdio.d

```diff
@@ -4932,6 +4958,20 @@ private struct ChunksImpl
     f.close();
 }
 
+// Issue 21730 - null ptr dereferenced in ChunksImpl.opApply (SIGSEGV)
+@system unittest
+{
+    import std.exception : assertThrown;
+    static import std.file;
+
+    auto deleteme = testFilename();
```

Comment:
Some windows computers fail here with:

```
std.file.FileException@std\file.d(1065): C:\Users\VSSADM~1\AppData\Local\Temp\deleteme.dmd.unittest.pid1124-детка.stdio.d.4967: The process cannot access the file because it is being used by another process.
```

I have little experience with `testFilename` and even less experience with windows. What mistake do I make here?

Reply:
My guess would be that the following happens:

1. The file is created and is held open.
2. The unit test attempts to delete the file. This fails, and throws an exception.
3. The `scope(exit)` block runs as part of exception unwinding. It then tries to delete the file again, which again throws the exception.

Reply:
Hey, thanks. I exchanged removing and closing. The original segfault still happens without the fix. Now I hope this works.